bats/fb-katello-client.bats

```diff
@@ -78,6 +78,21 @@ load fixtures/content
   done
 }
 
+@test "try fetching container content" {
+  tPackageExists podman || tPackageInstall podman
+  podman login $HOSTNAME -u admin -p changeme
+  CONTAINER_PULL_LABEL=`echo "${ORGANIZATION_LABEL}-${PRODUCT_LABEL}-${CONTAINER_REPOSITORY_LABEL}"| tr '[:upper:]' '[:lower:]'`
```

Comment:
unrelated to this PR, but it feels like this should live in https://github.com/theforeman/forklift/blob/master/bats/fixtures/content.bash?